### PR TITLE
Missing whitespace in line 1

### PR DESCRIPTION
**Pull request by CMarah**

### CODE REVIEW

**Comment by gbandres98** [comment]
After reviewing the changes I think this can be merged

**Comment by gbandres98** [comment]
Also missing from the changelog, a tilde has been fixed in line 4

File: index.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Libro1: Introducción
+Libro 1: Introducción
 Libro 2: Fundamentos
 Libro 3: Metodología
-Libro 4: Aplicaciónes
\ No newline at end of file
+Libro 4: Aplicaciónes
```
Comment (gbandres98) [inline index.txt]: Typo, should be "Aplicaciones"

**Review by gbandres98 — CHANGES_REQUESTED** [review]

File: index.txt (hunk 1 above)
Comment (CMarah) [reply index.txt]: Great point! Merging to main branch, performing fast forward :)

**Comment by gbandres98** [comment]
Thanks Carlos! LGTM!

**Pull request closed without merging**